docs/docs/articles/gitlab.md

```diff
@@ -23,9 +23,11 @@ variables:
 
 setup-testkube:
   stage: setup
-  image: kubeshop/testkube-cli
   script:
+    - echo "Installing Testkube..."
+    - curl -sSLf https://get.testkube.io | sh
```

Comment:
@emamihe, did you test it? Based on the Discord reports the default `ruby` image doesn't have required `jq`, `kubectl` nor `helm` binaries.